packages-web/badge-button/src/BadgeButton.xml

```diff
@@ -1,89 +1,48 @@
 <?xml version="1.0" encoding="utf-8"?>
-<widget id="com.mendix.widget.custom.badgebutton.BadgeButton" needsEntityContext="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../node_modules/mendix/custom_widget.xsd">
-    <name>Badge button</name>
+<widget id="com.mendix.widget.web.badgebutton.BadgeButton" pluginWidget="true" offlineCapable="true" xmlns="http://www.mendix.com/widget/1.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.mendix.com/widget/1.0/ ../../../node_modules/mendix/custom_widget.xsd">
+    <name>Badge button v2</name>
```

Comment:
Do we want to display the version in the name of the widget?

Reply:
Yes, defined by Andries